CONTRIBUTORS.txt

```diff
@@ -63,9 +73,11 @@ needs to be listed here.
 - Jason Toffaletti <jtoffaletti@apple.com>
 - Jim Dovey <jimdovey@mac.com>
 - Joachim M. <joachim.me86@gmail.com>
+- Joakim Hassila <hassila@users.noreply.github.com>
 - Joe Smith <yasumoto7@gmail.com>
 - Johannes Weiss <johannesweiss@apple.com>
 - John Connolly <connoljo2@gmail.com>
+- John Connolly <jconnolly@touchbistro.com>
```

Comment:
@John-Connolly which one of the two is your preferred email address for the contributors list?

Reply:
```connoljo2@gmail.com``` is great. Thanks, @weissi!